iac/provider-gcp/nomad-cluster/nodepool-client.tf

```
@@ -59,7 +59,14 @@ resource "google_compute_region_autoscaler" "client" {
     mode = "ONLY_SCALE_OUT"
 
     cpu_utilization {
-      target = 0.6
+      target = var.client_cluster_autoscaling_cpu_target
+    }
+
+    metric {
+      name   = "agent.googleapis.com/memory/percent_used"
+      type   = "GAUGE"
+      filter = "resource.type = \"gce_instance\""
```

Also we should compare it only to used memory
```suggestion
      filter = "resource.type = \"gce_instance\" AND metric.labels.state = \"used\""
```

---

👍🏽